serverless/serverless-support.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 include::modules/serverless-getting-support.adoc[leveloffset=+1]
 
-[id="serverless-support-gather-info_{context}"]
+[id="serverless-support-gather-info"]
```

Comment:
Just curious why you removed the `_{context}` separator here? I see the variable is still defined in the assembly (line 4). (I confess that I don't fully understand how we use this variable.)

Reply:
Afaik it's just needed in modules, not assembly ID headings, but I could be wrong?

Reply:
The templates for mod docs don't specify the need to have an anchor for headings in assemblies. @openshift/team-documentation - do we need to specify this?

Reply:
Nope, the _{context} part should only be needed for modules.